src/utils/decoder.cr

```diff
@@ -54,6 +54,10 @@ module Mint
       id(node.name)
     end
 
+    def generate(node : TypeChecker::Variable)
+      raise "WTF"
```

Comment:
debug leftover? ;)

Reply:
kind of ;) it's not finished yet :smile: